walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,53 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Set;
+import java.util.ArrayList;
 
 public final class FindMeetingQuery {
+
+  private boolean attendeeOverlap(Collection<String> requestAttendees, Set<String> eventAttendees){
+    for(String attendee : requestAttendees) {
```

Comment:
this implementation could be simpler if requestAttendees were also a Set<String>